paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletScreen.kt

```diff
@@ -563,16 +554,11 @@ private fun AddPaymentMethodRow(
             .clickable(enabled = isEnabled, onClick = onAddNewPaymentMethodClick),
         verticalAlignment = Alignment.CenterVertically
     ) {
-        Icon(
-            painter = painterResource(R.drawable.stripe_link_add_green),
-            contentDescription = null,
-            modifier = Modifier.padding(start = HorizontalPadding, end = 12.dp),
-            tint = Color.Unspecified
-        )
+        Spacer(modifier = Modifier.width(HorizontalPadding))
         Text(
             text = stringResource(R.string.stripe_add_payment_method),
             modifier = Modifier.padding(end = HorizontalPadding),
```

Comment:
Can we do this instead of adding the `Spacer` above?
```suggestion
            modifier = Modifier.padding(horizontal = HorizontalPadding),
```